engine/src/engine.lcb

```diff
@@ -181,6 +183,7 @@ end syntax
 /**
 Summary:	The property of a script object.
 Property: 	The name of the property to manipulate
+Set:		The name of the custom property set to access
```

Comment:
@livecodeian I don't think a single doc for multiple syntax blocks is supported by the docs parser. I don't think it will show the syntax properly in the dictionary so you may need to duplicate the docs here for the two variants.